lib/learn_open/environments.rb

```diff
@@ -0,0 +1,27 @@
+module LearnOpen
+  module Environments
+    def self.classify(options)
+       environment_vars = options.fetch(:environment_vars, LearnOpen.environment_vars)
+       platform         = options.fetch(:platform, LearnOpen.platform)
+       if jupyter_container?(environment_vars)
+         JuptyerContainerEnvironment.new(options)
+       elsif ide_environment?(environment_vars)
+         IDEEnvironment.new(options)
```

Comment:
Why `IDEEnvironment` vs `JupyterContainer`? Isn't the `IDEEnvironment` also a container?

Reply:
Not quite. A Jupyter container has less installed. The IDE has stuff like Java installed. Also, the IDE can't open jupyter labs well.